tjreports/site/views/reports/view.base.php

```diff
@@ -189,7 +189,11 @@ public function processData($type = 'html')
 		$this->headerLevel     = $this->model->headerLevel;
 
 		// Array_key - defaultColToHide column are present then get the key as value.
-		$defaultColToHide       = (array) $this->model->getState('defaultColToHide');
+		if ($this->model->getState('defaultColToHide'))
+		{
+			$this->defaultColToHide = array_keys($this->model->getState('defaultColToHide'));
```

Comment:
change as 
```if (!empty($this->model->getState('defaultColToHide')))```

`(array)` or a`rray_keys` what is working here? @ChavanAnkitaMahesh

Reply:
if we change as  this "if (!empty($this->model->getState('defaultColToHide'))) " or if (!empty(array($this->model->getState('defaultColToHide')))) or  (!empty(array_keys($this->model->getState('defaultColToHide'))))  it dosent work